### PR TITLE
Connect react redux

### PR DESCRIPTION
**Pull request by Samuellekpor**
In this feature, we:

1. Connect the app to the redux store
2. Add 'add new book', 'delete a book' and 'check status' functionalities.

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @Samuellekpor 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linter checks, Kindly fix the linter errors 🙏

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Samuellekpor** [comment]
Hello @Davidosky007 👋🏾 
Thanks for the quick reply. I am currently having some issues with the linters and I tried solving them but it didn't work. The Eslint is still giving me errors on GitHub even though everything is clean locally. I don't know what to do as I didn't want to disable the linter. I need help, please.

**Comment by Davidosky007** [comment]
> Hello @Davidosky007 👋🏾 Thanks for the quick reply. I am currently having some issues with the linters and I tried solving them but it didn't work. The Eslint is still giving me errors on GitHub even though everything is clean locally. I don't know what to do as I didn't want to disable the linter. I need help, please.

Hi 👋 @Samuellekpor Kindly add uppercases to the file you are importing for example as highlighted in the screenshot below you need to make your links like this as the Books.js file is in uppercase so when importing you have to use uppercase too  `../redux/books/Books,` `../redux/categories/Categories` try this and push your changes 👍 
image
image

**Comment by Samuellekpor** [comment]
Hello @Davidosky007 !!! 👋🏾 
Thank you for the suggestion. I did so and it worked 👍🏾 . I couldn't think about that because the file names in the redux folder are all lowercase. Thanks once again for the help. I really appreciate 🙏🏾

**Comment by rloterh** [comment]
Hi Sam, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with a ReadMe that does not follow the [standardized ReadMe template](https://github.com/microverseinc/readme-template). Kindly ensure to update your ReadMe to this [standardized template](https://github.com/microverseinc/readme-template), and then request another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,150 +1,201 @@
...
 
+After you're finished please remove all the comments and instructions!
+-->
 
-## Get Started
+<div align="center">
 
-To get the content of this project locally you need to run the following commands in your terminal.
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (gemmen29) [inline README.md:28]: - [ ] [OPTIONAL] Kindly add this image to your repo to appear in your README file.

**Review by gemmen29 — APPROVED** [review]
Hi @Samuellekpor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Samuellekpor) [reply README.md:28]: Hello @gemmen29 !!!
Thanks for the review. I kept the image in the README file. I didn't remove it.

**Pull request merged**